README.md

```diff
@@ -431,6 +431,8 @@ ANSWERS.addComponent('SearchBar', {
   clearButton: true,
   // Optional, redirect search query to url
   redirectUrl: 'path/to/url',
+  // Optional, target frame for the redirect url, defaults to current frame
+  redirectUrlTarget: '_self',
```

Comment:
Do you think it would be helpful to list possible options for the target such as "_parent", "_blank" or "_top"?